.github/workflows/release.yaml

```diff
@@ -3,6 +3,7 @@ on:
     # Sequence of patterns matched against refs/tags
     tags:
     - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+    - 'knative-v*' # Push events to matching knative 1.x tags format, i.e knative-v20.15.10
```

Comment:
nit: I'd rather have this part of a separate PR (or the release notes docs PR) since the title and description is only about `ko` here but we're also introducing a different behavior of the workflow.

Reply:
This is my bad let me remove it cause we are not going to use that release format to run release actions (at least yet)